gemrb/core/GameScript/GameScript.cpp

```diff
@@ -1821,8 +1819,10 @@ Script* GameScript::CacheScript(const ResRef& resRef, bool AIScript)
 	if (!stream) {
 		return NULL;
 	}
-	stream->ReadLine( line, 10 );
-	if (strncmp( line, "SC", 2 ) != 0) {
+	
+	std::string line;
+	stream->ReadLine(line, 10);
+	if (line.compare(0, std::string::npos, "SC") != 0) {
```

Comment:
Why not just use operator== in such cases?

Reply:
ah, yes, an oversight on my part certainly.